extensions/mssql/package.nls.json

```diff
@@ -178,5 +178,9 @@
 	"title.newTable": "New Table",
 	"title.designTable": "Design",
 	"mssql.parallelMessageProcessing" : "[Experimental] Whether the requests to the SQL Tools Service should be handled in parallel. This is introduced to discover the issues there might be when handling all requests in parallel. The default value is false. Relaunch of ADS is required when the value is changed.",
-	"mssql.tableDesigner.preloadDatabaseModel": "Whether to preload the database model when the database node in the object explorer is expanded. When enabled, the loading time of table designer can be reduced. Note: You might see higher than normal memory usage if you need to expand a lot of database nodes."
+	"mssql.tableDesigner.preloadDatabaseModel": "Whether to preload the database model when the database node in the object explorer is expanded. When enabled, the loading time of table designer can be reduced. Note: You might see higher than normal memory usage if you need to expand a lot of database nodes.",
+
+	"mssql.objectExplorer.groupBySchema": "When enabled, the database objects in Object explorer will be categorized by schema.",
```

Comment:
Probably should be "Object Explorer" (`e` capitalized) since this is a proper noun